File: Aula21_26_11_2020/Py/letraL.py
soma = 0

for i in range(0, 15):
    i = i + 1
    num = int(input("Digite o valor do número {}: ".format(i)))

    if num > 0:
        f = 1
        cont = 1

        while cont <= num:
            f *= cont
            cont = cont + 1
        soma = soma + f

print("A soma dos fatoriais desses números é igual a {}".format(soma))

File: Aula21_26_11_2020/Cs/letraL.cs
// using System;

// namespace Cs
// {
//     class LetraL
//     {
//         static void Main(string[] args)
//         {
//             int i, num, f, cont, soma;
//             soma = 0;

//             for (i = 1; i <= 15; i++)
//             {
//                 Console.Write($"Digite o valor do número {i}: ");
//                 num = Convert.ToInt32(Console.ReadLine());

//                 if (num > 0)
//                 {
//                     f = 1;
//                     cont = 1;
            
//                     while(cont <= num)
//                     {
//                         f *= cont;
//                         cont++;
//                     }
//                     soma += f;
//                 }
//             }
            
//             Console.WriteLine($"A soma dos fatoriais desses números é igual a {soma}");
//         }
//     }
// }